.github/workflows/CI.yml

```diff
@@ -72,6 +72,12 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@v1
+      - name: Build libclangex
+        if: ${{ matrix.libclangex == 'local' }}
+        run: |
+          julia --project=deps -e 'using Pkg; Pkg.instantiate()'
+          julia --project=deps deps/build_local.jl
+          cp LocalPreferences.toml test/
```

Comment:
We might need to add another step for running Clang.jl's generator script for updating bindings.

Reply:
Yeah, we probably should. Also we need to bifurcate the bindings we generate.